packages/react-conformance/src/defaultErrorMessages.tsx

```diff
@@ -12,312 +10,244 @@ import * as path from 'path';
-        ) +
-          resolveErrorMessages([
-            `Make sure that ${resolveInfo(fileName)} contains ${resolveInfo(
-              displayName + `.displayName = '` + displayName + `'`,
-            )}`,
-          ]) +
-          receivedErrorMessage(error),
-      );
-    }
-    failedTests.push('component-has-displayname');
+    // 1. Make sure the component file contains "displayName".displayName = '"displayName"'
+    return getErrorMessage({
+      displayName,
+      overview: `has an incorrect display name. It received: ${testErrorInfo(
+        componentDisplayName.replace('Styled', ''),
+      )}`,
+      suggestions: [
+        `Make sure the component file contains ${resolveInfo(`${displayName}.displayName = '${displayName}'`)}`,
+      ],
+      error,
+    });
   },
 
-  'component-handles-ref': (componentInfo: ComponentDoc, testInfo: IsConformantOptions, error: string) => {
+  'component-handles-ref': (testInfo: IsConformantOptions, error: Error) => {
     const { displayName } = testInfo;
     const { resolveInfo } = errorMessageColors;
 
     // Message Description: Handles scenario where the ref is not defined when passed to the component.
     //
-    // It appears that "displayName" doesn't have a defined ref value.
+    // It appears that "displayName" doesn't handle the ref prop.
     // To resolve this issue:
-    // 1. Check if your component has a ref.
-    // 2. If you component is a function component make sure that your are using forwardRef.
+    // 1. Check if your component has a ref prop.
+    // 2. For function components, make sure that you are using forwardRef.
     // 3. Check if your component passes ref to an inner component and add targetComponent to isConformant in
     //    your test file.
-    console.log(
-      defaultErrorMessage(`component-handles-ref`, displayName, `defined ref value.`) +
-        resolveErrorMessages([
-          `Check if your component has a ${resolveInfo('ref')}.`,
-          `If you component is a function component make sure that your are using ${resolveInfo('forwardRef')}.`,
-          `Check if your component uses an element ref and add ${resolveInfo(
-            `elementRefName: 'elementRef'`,
-          )} to isConformant in your test file.`,
-        ]) +
-        receivedErrorMessage(error),
-    );
+    // 4. If your component uses an "elementRef" prop instead of "ref", add elementRefName: 'elementRef' to
+    //    isConformant in your test file.
+    return getErrorMessage({
+      displayName,
+      overview: `doesn't handle the "ref" prop.`,
+      suggestions: [
+        `Check if your component has a ${resolveInfo('ref')} prop.`,
+        `For function components, make sure that you are using ${resolveInfo('forwardRef')}.`,
+        `Check if your component passes the ref to an inner component and add ${resolveInfo('targetComponent')} ` +
+          'to isConformant in your test file.',
```

Comment:
```suggestion
        `Check if your component passes the ref to an inner component and add ${resolveInfo('targetComponent')} 
        to isConformant in your test file.`,
```

Reply:
That will introduce a line break and a bunch of spaces in the final message :(